introduction.md

```diff
@@ -1 +1 @@
-I love Ruby and JavaScript!
\ No newline at end of file
+On my way to becoming a GitHub master. I love Ruby and JavaScript!\n \n* Git \n* CSS
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?